3.6/root/opt/app-root/etc/scl_enable

```diff
@@ -2,5 +2,8 @@
 #            doing. This file is sourced everytime the shell session is opened.
 # This will make scl collection binaries work out of box.
 unset BASH_ENV PROMPT_COMMAND ENV
-source scl_source enable httpd24 $NODEJS_SCL rh-python36
+if head "/etc/redhat-release" | grep -q "^CentOS Linux release 7" || \
+   head "/etc/redhat-release" | grep -q "^Red Hat Enterprise Linux release 7"; then
```

Comment:
Needs to be `grep -q "^Red Hat Enterprise Linux Server release 7"`

Reply:
Likely the culprit for the rhel7 failures

Reply:
Oh, right! I think some instances have it without the Server though, so I'll try to do both.